### DISCUSSION TITLE
[Code]: Auto deselect files when clicking a single file to match normal MacOS user experience

### DISCUSSION THREAD

**Discussion by JamesSteiningerSciton** [mikekazakov/nimble-commander#351] [Ideas]
### Task Type

Refactoring

### Description

Extremely annoying to have to re-select a bunch of files in order to deselect them. Why doesn't this just match normal MacOS conventions?

**Comment by mikekazakov** [comment]
It was a deliberate decision to diverge from the typical MacOS UX here and instead to follow the UX of orthodox dual-pane file managers.
To deselect items you can use a Opt+Cmd+A hotkey.

**Reply by djdakta** [reply]
The Opt+Cmd+A deselects "all" the files at once.
